github/github_test.go

```diff
@@ -2206,3 +2206,36 @@ func TestRateLimits_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestParseTokenExpiration(t *testing.T) {
+	tests := []struct {
+		res    *http.Response
+		header string
+		want   Timestamp
+	}{
+		{
+			res: &http.Response{
+				Request: &http.Request{},
+				Header:  http.Header{},
+			},
+			header: "",
+			want:   Timestamp{},
+		},
+		{
+			res: &http.Response{
+				Request: &http.Request{},
+				Header:  http.Header{},
+			},
+			header: "2021-09-03 02:34:04 UTC",
```

Comment:
Can you please add one more test with a completely garbage header and show how it behaves in that case?